components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -267,6 +267,9 @@
     // Property to define the allowed scopes.
     private List<String> allowedScopes = new ArrayList<>();
 
+    // Property to define the allowed scopes.
```

Comment:
Please change the comment as a description of the attribute unsupportedIntrospectionClaims